src/d3d9/d3d9_swapchain.cpp

```diff
@@ -177,12 +177,9 @@ namespace dxvk {
     if (unlikely(dstTexInfo->Desc()->Pool != D3DPOOL_SYSTEMMEM && dstTexInfo->Desc()->Pool != D3DPOOL_SCRATCH))
       return D3DERR_INVALIDCALL;
 
-    VkExtent3D dstTexExtent = dstTexInfo->GetExtentMip(dst->GetMipLevel());
-    VkExtent3D srcTexExtent = srcTexInfo->GetExtentMip(0);
-
-    dstTexInfo->CreateBufferSubresource(dst->GetSubresource(), dstTexExtent.width > srcTexExtent.width || dstTexExtent.height > srcTexExtent.height);
-    Rc<DxvkBuffer> dstBuffer = dstTexInfo->GetBuffer(dst->GetSubresource());
-    Rc<DxvkImage>  srcImage  = srcTexInfo->GetImage();
+    dstTexInfo->CreateBuffer(true);
```

Comment:
This always initializes now, ugh.

Reply:
Its not the biggest deal, but deserving of a comment for us to come back to.

Reply:
what's the problem with that exactly?

Reply:
The problem is that we're memsetting memory which will be overwritten immediately.

I changed it to only initialize if the texture has more than one mip level.
That should cover the majority of cases.

Reply:
You are missing the `dstTexExtent.width > srcTexExtent.width || dstTexExtent.height > srcTexExtent.height` check that used to be there.

Reply:
Done.